MapleServer2/Enums/SystemNotice.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace MapleServer2.Enums;
 
-public enum SystemNotice : int
+public enum SystemNotice
```

Comment:
Like this and many others, has this PR even been tested? Without indicating it's an int, it's going to ruin a lot of packets being written. How will the server know when to write an int or not?

Reply:
by default its an int

Reply:
Yes this has been tested and Rider says this: 'int' is default enum governing type

Reply:
the compiler assumes its an int by default thats why its ok to remove it